### PR TITLE
TW-1147 Improve getting exchange rates

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -166,28 +166,18 @@ app.get('/api/abtest', (_, res) => {
...
   const { data: tezExchangeRate, error: tezExchangeRateError } = await getProviderStateWithTimeout(
     tezExchangeRateProvider
   );
-  if (tokensExchangeRatesError !== undefined) {
-    return res.status(500).send({
-      error: tokensExchangeRatesError.message
-    });
-  } else if (tezExchangeRateError !== undefined) {
+
+  if (tezExchangeRateError !== undefined) {
```
Comment (lendihop) [inline src/index.ts:174]: `if (isDefined(tezExchangeRateError))`

**Review by lendihop — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (keshan3262) [reply src/index.ts:174]: Sorry, it caused a type error

**Review by lendihop — APPROVED** [review]

File: src/utils/tokens.ts [hunk 2]
```diff
@@ -18,31 +20,74 @@ import {
...
 const THREE_ROUTE_TEZ_SYMBOL = 'XTZ';
-const PROBE_TEZ_AMOUNT = 10;
-const probeSwapsProvider = new DataProvider(Infinity, async (outputTokenSymbol: string) => {
+const EMPTY_SWAP = {
+  input: 0,
+  output: 0,
+  chains: []
+};
+
+const ASPENCOIN_ADDRESS = 'KT1S5iPRQ612wcNm6mXDqDhTNegGFcvTV7vM';
```
Comment (lourenc) [inline src/utils/tokens.ts:55]: I think it's safe to remove this custom logic for this coin – it doesn't seem to be used or transferred at all

**Review by lourenc — APPROVED** [review]
Approved, but please see one comment

File: src/utils/tokens.ts (hunk 2 above)
Comment (keshan3262) [reply src/utils/tokens.ts:55]: You're right, the last transfer was on January 31st

**Review by lendihop — APPROVED** [review]

**Review by NovakVlad2424 — APPROVED** [review]
done

**Review by IrynaKhyzhynska — APPROVED** [review]
done

**Pull request merged**